packages/app/src/app/components/CreateSandbox/Import/Import.tsx

```diff
@@ -179,23 +153,38 @@ export const Import: React.FC<ImportProps> = ({ onRepoSelect }) => {
       >
         Enter the GitHub repository URL to import
       </Text>
-      <form onSubmit={handleFormSubmit}>
+      {hasMaxPublicRepositories ? <MaxPublicRepos /> : null}
+      <Element
+        {...(hasMaxPublicRepositories
+          ? {
+              as: 'div',
+              css: {
+                opacity: hasMaxPublicRepositories ? 0.4 : 1,
+                pointerEvents: hasMaxPublicRepositories ? 'none' : 'initial',
+              },
+            }
+          : { as: 'form', onSubmit: handleFormSubmit })}
+      >
         <Stack gap={2}>
           <Input
+            aria-disabled={hasMaxPublicRepositories}
             aria-describedby="form-title form-error"
             aria-invalid={Boolean(url.error)}
-            onChange={handleUrlInputChange}
+            css={{ height: '32px' }}
             placeholder="GitHub Repository URL"
+            tabIndex={hasMaxPublicRepositories ? -1 : 0}
```

Comment:
I think if we use `aria-disabled` instead of `disabled` the goal is that the element is still tabbable, but removing `tabIndex` would nullify this? I think it's fine in this case to use `disabled`, or is the "disabled state" described by `aria-describedby` in this case?

Reply:
@tristandubbeld I didn't want to override the css that gets applied when the button is disabled, so I went for aria-disabled and tabIndex to have the semantics without the UI.